sandpack-react/src/components/Preview/index.tsx

```diff
@@ -34,6 +34,7 @@ export interface PreviewProps {
   showRefreshButton?: boolean;
   showSandpackErrorOverlay?: boolean;
   actionsChildren?: JSX.Element;
+  children?: JSX.Element;
```

Comment:
Let's use `React.FC instead, in order to keep the same format for all component with children